src/log.js

```diff
@@ -198,20 +200,13 @@ class Log extends GSet {
    * @return {Log}   New Log containing the appended value
    */
   async append (data, pointerCount = 1) {
```

Comment:
Your previous comment was a good point though. I was thinking if we should make it (calling `canAppend` or `sign`) explicit and call it in the log instead of in Entry?

Reply:
I already forgot it a second time :joy: so we could either make it explicit, or make a generalized testing suite for the access controller API to ensure that all access controllers conform to this API.